jf_agent/main.py

```diff
@@ -143,6 +143,29 @@ def main():
                     print(f'Error: Unable to access explicitly-included project {proj}.')
                     return
 
+        # Memory & Disk Usage diagnostics
+        import os
+        import shutil
+
+        print("Memory & Disk Usage:")
+        # From here:
+        # https://stackoverflow.com/questions/51248144/getting-cpu-and-memory-usage-of-a-docker-container-from-within-the-dockerized-ap
+        used = int(
+            os.popen('cat /sys/fs/cgroup/memory/memory.usage_in_bytes').readlines()[0].strip()
```

Comment:
wondering if we want to wrap all these os calls in a try/catch -- probably not hard to imagine something going wrong, and I imagine we don't want to kill the entire agent run